Jenkinsfile

```diff
@@ -282,6 +282,21 @@ pipeline {
         githubNotify(context: "${env.GITHUB_CHECK_ITS_NAME}", description: "${env.GITHUB_CHECK_ITS_NAME} ...", status: 'PENDING', targetUrl: "${env.JENKINS_URL}search/?q=${env.ITS_PIPELINE.replaceAll('/','+')}")
       }
     }
+    stage('Stable') {
+      options { skipDefaultCheckout() }
+      when {
+        branch 'master'
+      }
+      steps {
+        deleteDir()
+        unstash 'source'
+        dir("${BASE_DIR}"){
+          setupAPMGitEmail(global: false)
+          sh(label: "checkout ${BRANCH_NAME} branch", script: "git checkout -f '${BRANCH_NAME}'")
+          gitCreateTag(tag: 'stable', tagArgs: '--force', pushArgs: '--force')
```

Comment:
We use something similar for the apm-pipeline-library:

- https://github.com/elastic/apm-pipeline-library/blob/c8f9802cda896d92701a67504c66b0f26fda9169/.ci/Jenkinsfile#L229-L241

Reply:
So with `tagArgs: '--force', pushArgs: '--force'` it would remove the former `stable` tag and set a new one?

Reply:
exactly, it does override the existing tag if exists.

```
       -f, --force
           Replace an existing tag with the given name (instead of failing)
```

We use the same approach to override the `current` tag in the jenkins shared library (this is the way we can deliver new feature without the need to raise PRs in each consumer)

Reply:
For instance:

![image](https://user-images.githubusercontent.com/2871786/113721432-70dbd900-96e7-11eb-8e27-35f9248730b4.png)